metricbeat/docs/fields.asciidoc

```diff
@@ -8762,6 +8762,14 @@ The disk name. For example: `/dev/disk1`
 
 
 [float]
+=== system.filesystem.device_type
```

Comment:
I'm thinking if we shouldn't name this simpler as `system.filesystem.type`. I usually think of `ext4` (for example) as a filesystem type, so that seems to fit. `device_type`, on the other had, makes me think of hardware.